src/app/features/settings/go-pro/services/go-pro-media.service.ts

```diff
@@ -250,7 +250,7 @@ export class GoProMediaService {
 
       const data = response.data;
 
-      const files = data.media[0].fs as any[];
+      const files = (data.media[0].fs as any[]).reverse();
```

Comment:
I noticed that files were showing oldest first. 
I could also read GoPro docs to get asc by timestamp. But I think just reverse array is quicker solution